docs/tutorials/node-app.rst

```diff
@@ -0,0 +1,143 @@
+Bundle a Node.js app into a ROCK
+********************************
+
+This tutorial describes the steps needed to bundle a typical Node.js application
+into a ROCK.
+
+Prerequisites
+-------------
+- snap enabled system (https://snapcraft.io)
+- LXD installed (https://linuxcontainers.org/lxd/getting-started-cli/)
+- skopeo installed (https://github.com/containers/skopeo)
```

Comment:
Since the tutorial uses the version of skopeo included in the rockcraft snap is this necessary?

Reply:
Nope! I copied this blindly from another tutorial
I fixed this here but I'll create an issue to put this common preamble somewhere that all tutorials can reference